app/controllers/api/calls_controller.rb

```diff
@@ -4,12 +4,23 @@ class Api::CallsController < ApplicationController
   skip_before_action :verify_authenticity_token, raise: false
 
   def create
-    service = CallCreator.new(call_params, tracking_params)
+    success = verify_recaptcha(action: params[:recaptcha_action], minimum_score: 0.5)
```

Comment:
Perhaps having the recaptcha gem now makes `app/services/recaptcha3.rb` redundant?

Reply:
@shivashankar-f22labs ☝️ fyi

Reply:
I'm proposing it as an alternative to our own implementation - it would mean less code for us to maintain.

Reply:
@osahyoun @eyko 
Code what we have now is very simple. just 75 lines of code only.

Second, we need bit customisation as well. 
   We need to log the response for tracking the score as we have no way for that so far.

Third, we need to raise exception especially to Raven / slack when score is less than 0.5 for quickly monitoring the failed transaction with score

For these reasons I like existing code itself.